pages/projects/life-journeys.js

```diff
@@ -0,0 +1,140 @@
+import { Layout } from "../../components/organisms/Layout";
+import Head from "next/head";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { useTranslation } from "next-i18next";
+import { useRouter } from "next/router";
+import { CallToAction } from "../../components/molecules/CallToAction";
+
+function HTMList({ tag = "ul", content, listClassName, liClassName }) {
+  const parseList = (content) =>
+    content
+      .split("*") // split the string on asterisks
+      .filter((item) => item) // filter out empty strings
+      .map((item, index) => (
+        <li className={liClassName} key={index}>
+          {item.trim()}
+        </li>
+      ));
+
+  return tag === "ul" ? (
+    <ul className={listClassName}>{parseList(content)}</ul>
+  ) : (
+    <ol className={listClassName}>{parseList(content)}</ol>
+  );
+}
+
+export default function LifeJourneys(props) {
+  const { t } = useTranslation("common", "dc");
```

Comment:
Technically, this should say `["common", "lj"]`, but it seems like it works anyway, which is probably because I had the wrong syntax and then all of the namespaces get loaded.

Reply:
Here's the reference for the syntax: https://react.i18next.com/latest/usetranslation-hook#loading-namespaces

Reply:
lol oops, I can fix that. Was probably when I was first switching over to using a separate JSON. `lj` is defined in `getStaticProps` so must have forgot to switch it up at the top as well.